Source/WebKit/Shared/libwpe/WebEventFactory.cpp

```diff
@@ -281,15 +285,24 @@ static WebKit::WebPlatformTouchPoint::TouchPointState stateForTouchPoint(int mai
 WebTouchEvent WebEventFactory::createWebTouchEvent(struct wpe_input_touch_event* event, float deviceScaleFactor)
 {
     auto type = WebEventType::NoType;
+    static HashMap<int32_t, int32_t> activeTrackingTouchPoints;
```

Comment:
This code is used on Windows so you can't use static objects with destructors because order of destruction is undefined and crashes often ensure.

```suggestion
    static NeverDestroyed<HashMap<int32_t, int32_t>> activeTrackingTouchPoints;
```

You'll have to #include <wtf/NeverDestroyed.h>

Reply:
Will do, thanks.